### DISCUSSION TITLE
Gitlab SSO in telport

### DISCUSSION THREAD

**Discussion by compgeniuses** [gravitational/teleport#37638] [Q&A] [answered]
i noticed that teleport has SSO functionalities support with github,

Iw as wondering if the same could also be offered with gitlab as well

**Comment by stevenGravy** [comment, marked as answer]
GitLab is supported for Enterprise and Enterprise Cloud. Please see https://goteleport.com/docs/access-controls/sso/gitlab/

**Reply by compgeniuses** [reply]
oh, ok, is gitlab more premium than github, they do the same job, others prefer gitlab to github
